integration_tests/fixtures/activitiesApi/getActivity.json

```diff
@@ -179,12 +179,21 @@
         "riskLevel": null,
         "minimumIncentiveLevel": null
       },
+      "scheduleWeeks": 1,
       "slots": [
         {
           "id": 5,
+          "weekNumber": 1,
           "startTime": "10:00",
           "endTime": "11:00",
-          "daysOfWeek": ["Tue"]
+          "daysOfWeek": ["Tue"],
```

Comment:
Is this right? Two Tuesday flags .. one in the daysOfWeek array, and one boolean? Maybe check with Lee, but I think the daysOfWeek list has replaced the booleans?

Reply:
This is how it's currently being returned from the API

Reply:
How odd - I might need to look into this, but fine for now, thanks.

Reply:
Aside from adding the `weekNumber` field I didn't make any other changes to this model.

I do agree though – I'm not sure why both the array and flags are needed.